extra/modules/fiftyone-devicedetection/src/main/java/org/prebid/server/hooks/modules/fiftyone/devicedetection/model/boundary/CollectedEvidence.java

```diff
@@ -0,0 +1,23 @@
+package org.prebid.server.hooks.modules.fiftyone.devicedetection.model.boundary;
+
+import com.iab.openrtb.request.Device;
+import lombok.Builder;
+
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * A set of information pieces that module can use for
```

Comment:
Nitpick: Please remove Javadocs. We are trying to remove them everywhere since 99% of the code and argument names are self-explanatory.

Reply:
✔️ Applied